apps/meteor/server/models/raw/EmailInbox.ts

```diff
@@ -12,4 +12,28 @@ export class EmailInboxRaw extends BaseRaw<IEmailInbox> implements IEmailInboxMo
 	protected modelIndexes(): IndexDescription[] {
 		return [{ key: { email: 1 }, unique: true }];
 	}
+
+	async setDisabledById(id: IEmailInbox['_id']): Promise<ModifyResult<IEmailInbox>> {
+		return this.findOneAndUpdate({ _id: id, active: true }, { $set: { active: false } }, { returnDocument: 'after' });
+	}
+
+	async create(emailInbox: IEmailInbox): Promise<InsertOneResult<IEmailInbox>> {
+		return this.insertOne(emailInbox);
+	}
+
+	async updateById(id: IEmailInbox['_id'], data: UpdateFilter<IEmailInbox>): Promise<ModifyResult<IEmailInbox>> {
+		return this.findOneAndUpdate({ _id: id }, data, { returnDocument: 'after', projection: { _id: 1 } });
+	}
```

Comment:
typing is wrong, right?
```suggestion
	async updateById(id: IEmailInbox['_id'], data: UpdateFilter<IEmailInbox>): Promise<IEmailInbox['_id']> {
		return this.findOneAndUpdate({ _id: id }, data, { returnDocument: 'after', projection: { _id: 1 } });
	}
```